app/views/partials/theme/simple/items/form.liquid

```diff
@@ -82,6 +82,27 @@
     <button class="mt-8 button" value="{{ path }}">Submit</button>
   </form>
 
+  <div class="container dashboard w-1/2 px-4">
+    <p class="text-2xl text-gray-900 ">Reviews</p>
+    <div class="recent-reviews">
+      {% for review in item.reviews %}
+        <div class="relative py-2 border-t border-blue-300">
+          <div class="flex flex-wrap mb-4">
+            <div class="rating">
+              <div class="rating-point" style="width: {{ review.rating | divided_by:5 | times:100 }}%"></div>
+            </div>
+            <span class="rating-score">{{ review.rating | round: 2 }}</span>
+          </div>
+          <p><b>{{ review.owner}}</b> : {{ review.comment }}</p>
+          <form action="/reviews/delete" method="post" class="strong" >
```

Comment:
you miss CSRF token, take a look at `app/views/partials/theme/simple/passwords/form.liquid` for example which includes csrf token via `{% include 'authenticity_token' %}`. If you do not include CSRF token, the session will be invalidated and user will be logged out